accelerator/core/core.c

```diff
@@ -581,3 +583,32 @@ status_t ta_get_iri_status(const iota_client_service_t* const service) {
 
   return ret;
 }
+
+status_t ta_update_iri_conneciton(ta_core_t* core) {
+  status_t ret = SC_OK;
+  for (int i = 0; i < MAX_IRI_LIST_ELEMENTS, core->ta_conf.host_list[i]; i++) {
+    // update new IRI host
+    core->ta_conf.host = core->ta_conf.host_list[i];
+    core->ta_conf.port = core->ta_conf.port_list[i];
+    core->iota_service.http.host = core->ta_conf.host_list[i];
+    core->iota_service.http.port = core->ta_conf.port_list[i];
+    ta_log_info("Try tp connect to %s:%d\n", core->ta_conf.host, core->ta_conf.port);
+
+    ret = ta_core_set(core);
```

Comment:
`ta_core_set` not only initialize the `iota_service`, but it will also initialize `pow`, `cache` and  `db`.
Is it possible that we do not repeatedly set the `iota_service`?
Add one `iota_service` for each host in the iri_list, then we only need to set them once, and won't affect other services.